.github/workflows/ci.yml

```diff
@@ -0,0 +1,60 @@
+name: CI
+
+on:
+  push:
+    branches:
+    tags:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    name: PHP v${{ matrix.php }} with phpstan ${{ matrix.phpstan }} and code_standart ${{ matrix.code_standart}}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      matrix:
+        include:
+        - { os: ubuntu-latest, php: 7.1, experimental: false}
+        - { os: ubuntu-latest, php: 7.1, phpstan: true, experimental: false}
+        - { os: ubuntu-latest, php: 7.2, experimental: false}
+        - { os: ubuntu-latest, php: 7.2, code_standart: true, experimental: false}
+        - { os: ubuntu-latest, php: 7.3, experimental: false}
+        - { os: ubuntu-latest, php: 7.4, experimental: false}
+        - { os: ubuntu-latest, php: 8.0, experimental: true}
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: "Installing php"
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php }}
+        extensions: curl,mbstring,xdebug
+        coverage: xdebug
+        tools: composer
+    - name: Show PHP version
+      run: php -v && composer -V
+    - name: Show Docker version
+      run: if [[ "$DEBUG" == "true" ]]; then docker version && env; fi
+      env:
+        DEBUG: ${{secrets.DEBUG}}
+    - name: Download Composer cache dependencies from cache
+      id: composer-cache
+      run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+    - name: Cache Composer dependencies
+      uses: actions/cache@v1
+      with:
+        path: ${{ steps.composer-cache.outputs.dir }}
+        key: ${{ matrix.os }}-composer-${{ hashFiles('**/composer.json') }}
+        restore-keys: ${{ matrix.os }}-composer-
```

Comment:
Assuming we're using composer 2 here, I don't think the caching is worth the bother.